From 98962eb6d679985696571262fb6b14f1182e7a8c Mon Sep 17 00:00:00 2001
From: Michael Straube <straube.linux@gmail.com>
Date: Thu, 4 Oct 2018 19:56:58 +0200
Subject: [PATCH] staging: rtl8188eu: make struct field of mlme_handler const
Git-commit: 98962eb6d679985696571262fb6b14f1182e7a8c
Patch-mainline: v4.20-rc1
References: FATE#326906

The strings in the uses of struct mlme_handler are constant.
Change the type of field str from 'char *' to 'const char *'.

Suggested-by: Joe Perches <joe@perches.com>
Signed-off-by: Michael Straube <straube.linux@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/staging/rtl8188eu/include/rtw_mlme_ext.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/rtl8188eu/include/rtw_mlme_ext.h b/drivers/staging/rtl8188eu/include/rtw_mlme_ext.h
index ade68af15e04..3e8a0ab46ff2 100644
--- a/drivers/staging/rtl8188eu/include/rtw_mlme_ext.h
+++ b/drivers/staging/rtl8188eu/include/rtw_mlme_ext.h
@@ -232,7 +232,7 @@ enum SCAN_STATE {
 
 struct mlme_handler {
 	unsigned int   num;
-	char *str;
+	const char *str;
 	unsigned int (*func)(struct adapter *adapt, struct recv_frame *frame);
 };
 
-- 
2.19.1

